portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,100 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+:root {
+  --primary: #0c4682;
+  --spacing-lg: 2rem;
+  --spacing-md: 1rem;
+  --spacing-sm: 0.5rem;
+  --spacing-xs: 0.25rem;
+  --border-style: thin black solid;
+  --max-body-width: 1080px;
+
+  font-family: 'Open Sans', sans-serif;
+}
+
+html, body {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  max-width: var(--max-body-width);
+  margin: auto;
+}
+
+h1, h2, h3 {
+  font-weight: 600;
+}
+
+header {
+  align-items: center;
+  padding: var(--spacing-md) var(--spacing-lg);
+}
+
+header > .contact-container {
+  display: grid;
+  grid-template-columns: auto 1fr;
+  grid-auto-rows: min-content;
+  gap: var(--spacing-sm);
+
+  padding: var(--spacing-xs) 0;
+  border-left: var(--border-style);
+  padding-left: var(--spacing-sm);
+}
+
+header > .contact-container > .entry {
+  display: contents;
+}
+
+header > .contact-container > .entry > .title {
+  font-weight: bold;
+}
+
+header > .title-container {
+  text-transform: uppercase;
+  margin: var(--spacing-md) 0;
 }
 
-#greeting-container {
-  margin-top: 20px;
+header > .title-container > .title {
+  color: var(--primary);
 }
+
+header > .title-container > :first-child {
+  margin-top: 0;
+  margin-bottom: var(--spacing-sm);
+}
+
+header > .title-container > :last-child {
+  margin-bottom: 0;
+  margin-top: var(--spacing-sm);
+}
+
+main {
+  padding: var(--spacing-md) var(--spacing-lg);
+}
+
+main > section:not(:last-child) {
+  margin-bottom: var(--spacing-md);
+}
+
+main > section > .title {
+  margin-top: 0;
+  border-bottom: var(--border-style);
+  color: var(--primary);
+}
+
+main > section > * {
+  margin-bottom: var(--spacing-sm);
+}
+
+main > section > .entry > .description > * {
+  margin: 0;
+}
+
+.space-between-flex {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.space-between-flex > :first-child {
+  margin-right: var(--spacing-md);
+}
```

Comment:
nit: can you add a newline to the end of the file?  it's just a convention, but GitHub yells at you:

![image](https://user-images.githubusercontent.com/1651960/83550653-2e4cdd00-a4d5-11ea-964e-9a384419eac1.png)

Reply:
Done!